### PR TITLE
fix(upgrade): create CLI configs

### LINKED ISSUE #19925: influxd upgrade does not create a configs file for working with the CLI
as part of the `influx setup` process, we create a `configs` file which stores a CLI configuration for accessing the instance from the command line tools. 

we need to add that to the upgrade command. I think it happens here: https://github.com/influxdata/influxdb/blob/master/cmd/influx/setup.go#L169

it is created in the `~/influxdbv2/configs` file by default

### PR DESCRIPTION
**Pull request by alespour**
Closes #19925

This PR contains code that creates CLI configs during upgrade from v1.

- [X] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [X] Rebased/mergeable
- [X] Tests pass
- [X] CHANGELOG.md updated with a link to the PR (not the Issue)

### CODE REVIEW

File: cmd/influxd/upgrade/upgrade.go [hunk 1]
```diff
@@ -149,6 +175,13 @@ func NewCommand() *cobra.Command {
 			Desc:    "path for boltdb database",
 			Short:   'm',
 		},
+		{
+			DestP:   &options.target.configsPath,
+			Flag:    "configs-path",
```
Comment (danxmoran) [inline cmd/influxd/upgrade/upgrade.go:180]: Could we rename the flag to be even more specific, i.e. `influx-configs-path`? I'm worried about confusion with the existing `config-file` argument.

File: cmd/influxd/upgrade/upgrade.go [hunk 2]
```diff
@@ -149,6 +175,13 @@ func NewCommand() *cobra.Command {
 			Desc:    "path for boltdb database",
```
Comment (danxmoran) [inline cmd/influxd/upgrade/upgrade.go:175]: It's too far away from the diff for me to comment on directly, but could you add a point `3` to the list at [this line](https://github.com/influxdata/influxdb/pull/19937/files#diff-e5c52e2be1a2771a344e5d0402468658d34a70fe9ba50453edb3b3b3e97a4eaeR149) describing the new behavior?

Comment (alespour) [reply cmd/influxd/upgrade/upgrade.go:176]: I hope I got this correctly (wasn't sure where the link point to).

Comment (danxmoran) [reply cmd/influxd/upgrade/upgrade.go:176]: Yup, that was it

**Review by danxmoran — APPROVED** [review]

File: cmd/influxd/upgrade/upgrade.go [hunk 3]
```diff
@@ -40,12 +41,36 @@ type configV1 struct {
 		Dir    string `toml:"dir"`
 		WALDir string `toml:"wal-dir"`
 	} `toml:"data"`
+	Http struct {
```
Comment (docmerlin) [inline cmd/influxd/upgrade/upgrade.go:44]: please don't build the url using string concatination.  Please use the `net.URL` object.

File: cmd/influxd/upgrade/upgrade_test.go [hunk 4]
```diff
@@ -6,6 +6,9 @@ import (
 	"path/filepath"
 	"testing"
 
+	"github.com/BurntSushi/toml"
+	"github.com/google/go-cmp/cmp"
+
```
Comment (docmerlin) [inline cmd/influxd/upgrade/upgrade_test.go:11]: remove this line please

**Review by docmerlin — CHANGES_REQUESTED** [review]

**Review by dgnorton — APPROVED** [review]

**Review by danxmoran — APPROVED** [review]
Re-LGTM

**Pull request merged**